pkg/sidecar/config.go

```diff
@@ -30,6 +30,12 @@ type Config struct {
 	SvidBundleFileNameDeprecated       string `hcl:"svidBundleFileName"`
 	RenewSignal                        string `hcl:"renew_signal"`
 	RenewSignalDeprecated              string `hcl:"renewSignal"`
+
+	// JWT configuration
+	JWTAudience string `hcl:"audience"`
```

Comment:
Should this be `jwt_audience`

Reply:
I think `jwt_audience` is better, I will change it